clusterloader2/testing/nichotr/config.yaml

```diff
@@ -0,0 +1,143 @@
+#Constants
+{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 56}}
```

Comment:
Let's set to 10 by default? IIRC, our doc says 1, 10, 30 for different scenarios.

Reply:
Sounds good